src/brogue/Rogue.h

```diff
@@ -2457,6 +2457,9 @@ typedef struct playerCharacter {
     enum NGCommands nextGame;
     char nextGamePath[BROGUE_FILENAME_MAX];
     uint64_t nextGameSeed;
+
+    // Path of the current save game or recording, NULL for a new game
+    char currentGamePath[BROGUE_FILENAME_MAX];
```

Comment:
Could you explain what this is for?

Reply:
It's for producing nice output when the game completes. It might be particularly useful if we support wildcards in the `-vn` path in future (as an end-to-end test)

Reply:
I should note that `nextGamePath` does exist but is zero-ed out once a game begins (which makes semantic sense since it is really only meant to be used for defining which game to load). Therefore currentGamePath is added to semantically capture the concept of the filename of the game in progress.